packages/react-vapor/src/components/form/Form.tsx

```diff
@@ -0,0 +1,29 @@
+import * as classNames from 'classnames';
+import * as React from 'react';
+
+type FormMods = 'mod-header-padding' | 'mod-form-top-bottom-padding';
+
+export interface IFormProps {
+    title?: string;
+    className?: string;
+    asCard?: boolean;

```

Comment:
Will you apply the suggestion of removing asCard and using a mod instead?

Reply:
I could but never got a response here: https://github.com/coveo/react-vapor/pull/1247#discussion_r322359734
Thoughts? :P

Reply:
The way I see it it modifies the way the form is displayed so it should be a mod

Eventually we could rename the class but it would be breaking for now